src/Operator/Operators.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Operator;
+
+use Twig\Operator\Binary\AbstractBinaryOperator;
+use Twig\Operator\Unary\AbstractUnaryOperator;
+
+class Operators implements \IteratorAggregate
```

Comment:
I suggest adding `@template-implements \IteratorAggregate<string, OperatorInterface>` to define the type of the iterator.